cmd/limactl/restart.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"github.com/lima-vm/lima/pkg/instance"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/spf13/cobra"
+)
+
+func newRestartCommand() *cobra.Command {
+	restartCmd := &cobra.Command{
+		Use:               "restart INSTANCE",
+		Short:             "Restart a running instance",
```

Comment:
Since `restartAction` only supports 0 or 1 argument, we should throw an error when the user runs `limactl restart foo bar baz` instead of silently ignoring the additional instances:

```go
Args: WrapArgsError(cobra.MaximumNArgs(1)),
```